localstack-core/localstack/services/kms/models.py

```diff
@@ -277,6 +277,20 @@ def key(self) -> RSAPrivateKey | EllipticCurvePrivateKey:
         )
 
 
+@dataclass
+class KeyRotationEntry:
+    key_id: str
+    key_material: bytes
+    key_material_id: str
+    key_material_state: str
+    rotation_date: datetime.datetime | None = None
+    rotation_type: str | None = None
+    key_material_description: str | None = None
+    expiration_model: str | None = None
+    import_state: str | None = None
+    valid_to: str | None = None
```

Comment:
Did you consider using the predefined class in `__init__.py` file which already includes this type `RotationsListEntry`?